### ISSUE TITLE
Handling configuration files

### ISSUE THREAD

**Issue by musjj** [Nudin/mplug#8]
mplug currently supports installing .conf template files from repos. But the problem comes when it's time to update your plugins.
It seems that currently, the update process mostly consists of git pulling the various plugin repos we have. But what if we have made changes to those .conf files? I'm not familiar with git, but I assume this will create a conflict.
What do you think?

**Comment by Nudin** [comment]
Yes, the updates are done by pulling the current state via git. If only the local version or only the upstream version of the file has changed this will work fine. If both, the user and the plugin author change the same file git will try to merge both versions, if however the same line was changed in both versions there will be a conflict.
This is not yet handled by mplug, the tool should probably ask the user about what to do.

**Comment by vnbubba** [comment]
Hello, I tried to uninstalled my plugin but my terminal show that:
mplug uninstall mpv-autosub
No environment variable found, guessing /home/hoang/.mpv as mpv config folder.
Not installed: mpv-autosub
How should i fix it?
